src/components/text/marked/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+import { Box, BoxProps, useMultiStyleConfig } from '@chakra-ui/react';
+
+import Highlight from './highlight';
+
+type SharedProps = Exclude<
+  BoxProps,
+  'position' | 'width' | 'display' | 'alignItems'
+>;
+
+type HighlightProps = SharedProps & {
+  variant?: 'highlight';
+  highlightColor?: 'white' | 'brand';
```

Comment:
maybe `brand.primary` for consistency in terms of passing a subset of colours form our pallette?